src/SubdomainInput/SubdomainInput.jsx

```diff
@@ -37,6 +37,7 @@ export const SubdomainInput = forwardRef(
       disabled,
       error,
       warning,
+      icRight,
```

Comment:
Please add prop type validation for `icRight`. Also, please check if we need a default value for it ( _"check"_ )to preserve backwards compatibility. I think it is used in Partners Portal only, but I'm not sure.

Reply:
22f2b6cf88a631cc4547a1dcc405c010023dfde5

It is used in the portal branding but the design does not have a "check" icon for it. No default prop is needed.